srcpkgs/scons/template

```diff
@@ -1,19 +1,18 @@
 # Template file for 'scons'
 pkgname=scons
-version=3.0.1
+version=3.0.2
 revision=1
 build_style=python2-module
-hostmakedepends="python"
-depends="python"
-noarch="yes"
-pycompile_module="SCons"
 make_install_args="--standard-lib --install-data=/usr/share"
-short_desc="A software construction tool"
+pycompile_module="SCons"
+hostmakedepends="python python-setuptools"
+depends="python"
+short_desc="Software construction tool"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.scons.org/"
 license="MIT"
+homepage="https://www.scons.org/"
 distfiles="${SOURCEFORGE_SITE}/$pkgname/$pkgname-$version.tar.gz"
-checksum=24475e38d39c19683bc88054524df018fe6949d70fbd4c69e298d39a0269f173
+checksum=d0afcf8f4c2ea5fa7af3c9321c0cb3ea1c83d137196be10c4cb2c79cc5dade01
 
 post_install() {
 	vlicense LICENSE.txt
```

Comment:
Right, the archive from sf does not contain a license file at all. Would it work to fetch the release tarball from github instead since vlicense is required for MIT and we don't vendor license files.

Reply:
Either that or package the license in files dir